ups/product_deps

```diff
@@ -250,7 +250,6 @@ gdmldir	product_dir	gdml
 ####################################
 product		version		qual	flags		<table_format=2>
 larg4		v10_00_03	-
-larsoft_data	v1_02_02	-
```

Comment:
Why remove larsoft_data?  Pretty sure that files there are needed by bits of larsim.

Reply:
Will check.

Reply:
@lgarren, I've verified that a `larsim` builds successfully standalone and all unit tests pass without `larsoft_data`.  If it turns out that a downstream package requires it, then we should add the dependency there.

Reply:
My concern is that some code may reference these files.  I doubt that the tests verify that, unfortunately.

Reply:
> My concern is that some code may reference these files. I doubt that the tests verify that, unfortunately.

You're right--just checked.  Will revert this change.